Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -352,7 +352,7 @@ final class CarbAbsorptionViewController: ChartsTableViewController, Identifiabl
                     )
 
                     if absorption.isActive {
-                        cell.observedValueTextColor = UIColor.COBTintColor
+                        cell.observedValueTextColor = UIColor.cobTintColor
```

Comment:
[nit] I know here you're following the naming that was present previously here, but is the "color" in the variable name redundant?

Reply:
iOS uses `tintColor`, so I will keep this as is.